waku/protocol/v2/waku_store.nim

```diff
@@ -173,11 +173,89 @@ proc encode*(rpc: HistoryRPC): ProtoBuffer =
   result.write(2, rpc.query.encode())
   result.write(3, rpc.response.encode())
 
+proc findIndex*(msgList: seq[IndexedWakuMessage], index: Index): int=
+  ## returns the position of an IndexedWakuMessage in msgList whose index value matches the given index
+  ## returns -1 if no match is found
```

Comment:
Could probably use Option type here instead of returning -1

Reply:
Thanks, changed